### ISSUE TITLE
Deactivating plugin -> blocks still available?

### ISSUE THREAD

**Issue by ajgw** [woocommerce/woocommerce-gutenberg-products-block#540]
I am bit surprised as I deactivated plugin version 2.0.1 and blocks still available. I did this because I had and have (in the past too) after upgrading plugin to a newer version errors in posts/pages were I had used Woocommerce blocks. Today after upgrading wc from 3.5.8 to 3.6.2 and in conjunction wc-blocks from 1.4.0 to 2.0.1 I checked posts with wc-blocks in them and yes they got a error -> could not recognized. So they marked with no value content. No resolving possible, only deleting and setting new up. In this case I had used products by category.

Beside I was wondering why blocks still available after deactivating blocks plugin? Any miracle?

**Comment by ryelle** [comment]
Hi @ajgw – the current blocks are included with WooCommerce core as of 3.6, so you got them when you updated. 

> I checked posts with wc-blocks in them and yes they got a error -> could not recognized.

That shouldn't be happening, can you copy or screenshot the error you're seeing?

**Comment by ajgw** [comment]
Hi @ryelle, well text in screenshot is german language:

![Bildschirmfoto 2019-04-25 um 16 42 45](https://user-images.githubusercontent.com/3390880/56744771-671a6e80-6779-11e9-8278-7cb18416b45a.png)

**Comment by ajgw** [comment]
This block contains unexpected or invalid content

**Comment by ryelle** [comment]
Can you open [the developer console](https://support.airtable.com/hc/en-us/articles/232313848-How-to-open-the-developer-console) and see if there are any red or yellow messages there, and if so, copy and paste them here?

**Comment by ajgw** [comment]
Well, I get a yellow warning at

`Block validation: Expected text `[products limit="3" columns="3" category="34" orderby="date" order="DESC"]`, saw `[products limit="16" columns="4" category="34" orderby="date" order="DESC"]`.`

which is related to blocks.min.js:2:99109

and too a red error at blocks.min.js:2:101370 and there this:

```
Expected:
<div class="wp-block-woocommerce-product-category">[products limit="3" columns="3" category="34" orderby="date" order="DESC"]</div>
```

```
Actual:
<div class="wp-block-woocommerce-product-category">[products limit="16" columns="4" category="34" orderby="date" order="DESC"]</div>
```

**Comment by ryelle** [comment]
@ajgw Thanks for the extra info. In this case, we changed how the default number of items in the rows & columns are configured– you can see the difference in the `limit` property. So you will have to re-configure your blocks now, but this is a one-time change and shouldn't happen again.

**Comment by ajgw** [comment]
Hi @ryelle I saw this curios behavior too. But accepting what you told me, that I have to live with behavior is not a good idea. I have twenty to thirty very different Woocommerce blocks in use and if I do not (and it is so that the error does not show me the kind of block it was before error) see which kind of block I have used, how do I replace it after then? Or shall I write down setting for each of them before I upgrade? No other solution possible?
